util/design/lib/OtpMemMap.py

```diff
@@ -20,6 +20,9 @@
 # the same seed for different classes)
 OTP_SEED_DIVERSIFIER = 177149201092001677687
 
+# This must match the rtl parameter ScrmblBlockWidth / 8
+SCRAMBLE_BLOCK_WIDTH = 8
```

Comment:
Since you're at it, can you use this constant and replace the hardcoded `8` in the alignment checks on lines 120 below?
```
    if check_int(part["offset"]) % 8:
        raise RuntimeError("Partition offset must be 64bit aligned")
    if check_int(part["size"]) % 8:
        raise RuntimeError("Partition size must be 64bit aligned")
```

Reply:
np